### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Fatima-hub333**
<h2> Math magicians: components </h2>

The following changes were implemented in this Milestone :Components: 

- [ ] Delete all the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called components.
- [ ] Inside components, create a new Calculator.js file.
- [ ] In Calculator.js, create a React component that matches this design:

- [ ] Once the Calculator is ready, import it and use it in your main component (App.js).

## Thanks for taking out the time to review this project 🧨
It really is appreciated :) ❤️ 
- Note: I personally love optional suggestions so please do leave some 😄

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  flex-direction: column;
+  width: 25%;
+  height: 100%;
+  margin: 7% auto;
+  border: 1px solid;
+  font-size: 20px;
 }
```
Comment (KDlamini) [inline src/App.css]: [OPTIONAL] You can optimize your UI by giving your calculator a grey border-color `#808080` to match the rest of the design. 
```
border: 1px solid #808080;
```

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,52 @@
...
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.orange-btn button {
+  background: orange;
+  width: 100%;
+  border: 1px solid;
 }
```
Comment (KDlamini) [inline src/App.css]: [OPTIONAL] You can do the same for your orange buttons as well.

Before:
image

After:
image

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+const Calculator = () => {
```
Comment (KDlamini) [inline src/components/Calculator.js]: According to the project requirements, the `Calculator.js` component should be class-based. Kindly refactor your component to use a class instead. You will get a chance to use functional components in the future, for now, it is important that you learn class-based components as well.

For example:
```
import React, { Component } from 'react';

class Calculator extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}
```

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Fatima-hub333 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Great PR description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Fatima-hub333) [reply src/App.css]: Thank you for taking the time to review my code. I gave a grey color to the border.

File: src/App.css (hunk 2 above)
Comment (Fatima-hub333) [reply src/App.css]: Thank you. Done with this required change.

File: src/components/Calculator.js (hunk 3 above)
Comment (Fatima-hub333) [reply src/components/Calculator.js]: Thank you for pointing out this mistake. I correct this one as well.

**Review by keroloussamy — APPROVED** [review]
Hi @Fatima-hub333, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**